### ISSUE TITLE
No Buttons for MultiRoot workspace

### ISSUE THREAD

**Issue by Luncher91** [mkloubert/vs-script-commands#4]
Hi,

Is there any chance to get script.commands parameters for the Multi-Root Workspace?

![grafik](https://user-images.githubusercontent.com/5971558/32648892-32206eec-c5f8-11e7-9696-d820758df6f6.png)

Best regards,
Andreas

**Comment by mkloubert** [comment]
currently not ... BUT I started to refactor the extension for supporting multi workspaces ... I wanted to wait for the official release before I finish implementation ... at this weekend I will do this for all my extensions including Script Commands

**Comment by Luncher91** [comment]
great to here, thank you. I am looking forward to your update 👍

**Comment by mkloubert** [comment]
@Luncher91 

Sorry, one week later as planned, but the **new version 6.0.0** should now be able to handle [multi workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces).

You can switch between worksapces by using the new `Select workspace` command:

![image](https://user-images.githubusercontent.com/4951215/32996720-a5883816-cd86-11e7-98a6-a3d05c5be811.png)

**Comment by Luncher91** [comment]
Don't worry :-) 
Great, thank you! 👍 works as expected!
Just an idea for improvement: determine in which workspace the currently selected file is and load the corresponding script.commands.

Nevertheless, should I close this issue ?

**Comment by mkloubert** [comment]
@Luncher91 

**I have released a new version 6.1.0, which is now able to reload the settings by active text editor automatically.**

For that, you should set `autoSelectWorkspace` to `(true)` in your **global settings** (`CTRL + ,`):

```json
{
    "script.commands": {
        "autoSelectWorkspace": true
    }
}
```

**Comment by Luncher91** [comment]
Wow, great! Thank you. Works as expected!
